app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_with  local: true do |f| %>
```

Comment:
Requirements state that this form needs to be created with a `form_for` [`REQUIRED`].

Reply:
that was deprecated, read the following documentation. https://guides.rubyonrails.org/form_helpers.html#using-form-for-and-form-tag

Reply:
Hello!  Rails documentation says that form_for is sfot deprecated and to not use it! Should we revert back to the deprecated form_for?

Reply:
"form_for. form_tag and form_for are soft deprecated and they will be replaced by form_with in the future."

Reply:
If it is soft deprecated then our guidelines need to be updated. Keep the `form_with` and don't change it.